app/controllers/clients_controller.rb

```diff
@@ -52,7 +52,7 @@ def set_client
     end
 
     def client_params
-      params.require(:client).permit(:name, :address, :phone, :website, :short_code)
+			params.require(:client).permit(:name, :address, :phone, :website, :short_code, :account_manager)
```

Comment:
make a post request (create). pass account_manager and test, that it is saved / exists (https://github.com/asm-products/mana-rails/blob/develop/spec/controllers/clients_spec.rb#L33)